docs/61_dandi_authentication.md

```diff
@@ -0,0 +1,83 @@
+# Work In Progress
+
+## Setting up your GitHub OAuth Account
+
+Please follow the steps in [Initialize Vendors - GitHub](../60_initialize_vendors/#github) to get started.
+
+For the next steps in setting up authentication, you'll want to record the values
+created during [Obtaining your Oauth App creds](..60_initialize_vendors/#obtaining-your-oauth-app-credentials).
+
+## Creating and Updating Objects in the DANDI Archive Admin Panel
+
+### Creating "Sites" and "Social App" Objects
+
+*In order to complete this step, you will need to have deployed an initial DANDI Archive API -- see here for more details [Creating the DANDI Archive API](../64_dandi_archive)
+
+First, sign in via the Django admin panel with your credentials created via the [Creating an admin user account for the DANDI Archive API step](../64_dandi_archive/#creating-a-django-superuser-admin-account)
+
+The Django Admin panel should be located at: `<your_full_api_url/admin`
+
+<br/><br/>
+<img
+src="../img/admin_panel.png"
+alt="admin_panel"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Next, you'll want to proceed to the `Sites' tab -- create 2 sites, one for your UI, and another for your API domain
+
+<br/><br/>
+<img
+src="../img/site_admin.png"
+alt="site_admin"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Next, you'll want to proceed to the `Social applications` tab, located on the left side panel. Once in this panel, you'll need to click
+`Add Social Application` in the top right corner
+
+<br/><br/>
+<img
+src="../img/social_app_panel.png"
+alt="social_app_panel"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Once in this panel, you'll want to create a `Social Application` object with:
+
+• The Provider as `GitHub`
+• The client ID as your GitHub OAuth client ID
+• The client secret as your GitHub OAuth client secret
+• Two `Sites` objects -- these objects you should have created a few steps prior
+
+### Updating (or creating) the "Application" Object
+
+Stay signed in the Django Admin panel. You'll next want to update (or create if not present) an `Application` object. Similar to the steps above,
+you'll need to populate the client ID with your Github OAuth client ID -- see the screenshot below for the configuration that would be appropriate.
+
+If this object exists (it can if you ran migrations first/deployed to Heroku), you can simply update the values to your appropriate redirect URLs and client ID.
```

Comment:
```suggestion
If this object exists (it can if you ran migrations first or deployed to Heroku), update the values to your appropriate redirect URLs and client ID.
```